### ISSUE TITLE
Cultists and Zryachi are extremely overtuned

### ISSUE THREAD

**Issue by January-0002** [Solarint/SAIN#53]
Something is incorrect on the cultist end of things in the 3.5.7 latest release. 

They shoot a LOT further than the maximum allowed distance. 

I sniped one at 250m and they shot back at that exact same distance (with accuracy that is also extremely overtuned), while their max view distance is 100m. 

Edit: expanding this issue to Zryachi, as he experiences similar unintended behavior. Will check for hostility on cultists by default and see what might be happening. Additional testing and recording is underway.

**Comment by January-0002** [comment]
Can confirm, Zriyachi can snipe past the preset view distance. Additionally, bosses seem to ignore those limits to a degree as well (inconclusive results)

**Comment by Solarint** [comment]
I'm not changing anything cultist specific. Is this still a thing that happens in new versions? Cultists are bullshit in live too.

**Comment by January-0002** [comment]
I will get some definitive testing going. Will update with results on this. While the mod does not impact any boss AI in specific (whether that be "boss" "cultist" or "zryachi"), it does indeed end up applying SOMETHING to them, as their behavior is not normal.

I'm expanding this issue to be as reported in the discord, as I have definitive evidence of at least Zryachi being broken when SAIN and BigBrain only are installed.

**Comment by January-0002** [comment]
Here are some findings that I found.

With SAIN, Zryachi enters his ZryachiFight AI layer the instant he sees the player at 390-410 metres. The testing distance was 400m at default SAIN settings (it should be capped at 180, as the setting says).
He is instantly hostile within that range and starts shooting.

Without SAIN, Zryachi STILL enters his ZryachiFight AI AI layer the instant he sees the player ar 390-410 metres, HOWEVER, he does not shoot unless provoked, or the bridge is tripped off. To confirm this, I have also spotted the sniper warning ONLY going when the bridge is reached, via the "Danger, Sniper" sign. 

SAIN may not be touching boss AI explicitly, however it affects them in some type of way.
